lib/active_admin/version.rb

```diff
@@ -1,3 +1,3 @@
 module ActiveAdmin
-  VERSION = '1.0.0.pre1'
+  VERSION = '1.0.0.pre2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.